app/models/miq_server.rb

```diff
@@ -487,6 +487,19 @@ def self.zone_is_modifiable?
     Zone.visible.in_my_region.count > 1
   end
 
+  def self.audit_vm_totals
```

Comment:
I'm thinking this auditing feature might be used by other things in the future, so lets get rid of the word "vm" from the name here and throughout.  Perhaps `audit_resource_totals`?

Reply:
or `audit_managed_resources` as I suggested in the comment for jobs.rb

Reply:
Ah yes, I like `audit_managed_resources` better

Reply:
Method name changed and pushed.